helm/templates/letsencrypt-issuer.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-issuer
+spec:
+  acme:
+    email: jdenen@pillartechnology.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource that will be used to store the account's private key.
+      name: hindsight-prod-tls-key
```

Comment:
Should `prod` be hardcoded into this name?

Reply:
I’d template it.

Reply:
I don't think its valuable to have the environment name in this key so to prevent unneeded complexity this will just be hindsight-tls-key in the respective environments.  It is just the private key for that environment.